### DISCUSSION TITLE
Command palette navigation refresh issue with releases

### DISCUSSION THREAD

**Discussion by brunobesson** [community/community#39927] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello there. I just started trying to play with the command palette, which is a nice tool indeed!

I noticed one edge case when switching from one repository to another: the `Releases` tab title isn't updated accordingly and stays "configured" for the previous repository. One has to refresh the page to make it correspond to the actual repository (or be hidden, if there is no release yet).

**Comment by ossanna16** [comment]
Hi there @brunobesson and welcome to our community! Thank you for the feedback!
